mlflow/requirements.txt

```diff
@@ -0,0 +1,2 @@
+mlflow==1.28.0
+boto3==1.24.62
```

Comment:
i believe requirements are lacking _psycopg2_ package, at least on my machine, composing the docker led up to `mlflow-mlflow-1  | ModuleNotFoundError: No module named 'psycopg2'` error.